bitnami/fluentd/templates/forwarder-daemonset.yaml

```diff
@@ -45,6 +45,10 @@ spec:
               value: {{ .Values.forwarder.configFile }}
             - name: FLUENTD_OPT
               value: {{ .Values.forwarder.extraArgs | quote }}
+            - name: FLUENTD_DAEMON_USER
+              value: {{ .Values.forwarder.daemonUser | quote }}
+            - name: FLUEND_DAEMON_GROUP
```

Comment:
typo, should be **FLUENTD**

Reply:
Thanks for letting us know! Would you mind creating a PR fixing the issue?

Reply:
If not we will create it ourselves but if you create it we would speed up the process.

Reply:
@javsalgar see https://github.com/bitnami/charts/pull/2838

Reply:
Thanks! We will check it

Reply:
apparently @irizzant is faster 👍

Reply:
Given you are replying here @javsalgar,  I'd like to also add another comment (slight relevant to the change here). We are facing issues start fluentd forwarder with root when we have PSP in place which has `allowedCapabilities` set to empty array.  So the start scripts (namely, `setup.sh`, `run.sh`) failed.  I had a look at the scripts, I felt confused as by default, the fluentd base folder is owned by `root:root`, I don't see why we need to do `chown` and `su` in the scripts when the user is already `root`. I get it, if there is no PSP, it would just be fine. But in this case, it does nothing if its already root. We had to patch it by mount volumes to override it. Do you think it makes more sense to just not do it when the user is `root`?

Reply:
Hi,

I think it would be better to create an issue instead, so other users can benefit from it. Do you mind moving it to a new issue?

Reply:
@javsalgar yea, sure, I will do it probably tomorrow or later. Thanks